cfme/fixtures/nuage.py

```diff
@@ -6,46 +8,64 @@
 
 
 def create_basic_sandbox(nuage):
-    box = {}
+    box = Namespace()
 
     # Create empty enterprise aka 'sandbox'.
-    enterprise = box['enterprise'] = nuage.create_enterprise()
-    logger.info('Created sandbox enterprise %s (%s)', enterprise.name, enterprise.id)
+    box.enterprise = get_object_dictionary(nuage.create_enterprise())
+    logger.info('Created sandbox enterprise %s (%s)',
+                box.enterprise['obj'].name, box.enterprise['obj'].id)
```

Comment:
I don't think this is a good or efficient pattern, `get_object_dictionary`. It odd to me in general, and combined with the use of argparse.Namespace makes me wonder what you're really trying to solve here.  I think collections.namedtuple is more appropriate in this context, as you define the object type and parameter names before using them. Namespace does give you more freedome to arbitrarily add attributes to `box`.

Back to my point - you've written and called `get_object_dictionary` to create a dictionary with `name` and `id` keys, and then right here after doing so you still directly reference `box.enterprise['obj'].name`, instead of `box.enterprise['name']`.  Both seem like awkward references, when `box.enterprise` _should_ be the object.